nx-X11/programs/Xserver/Imakefile

```diff
@@ -292,7 +292,7 @@ NXAGENTOBJS = hw/nxagent/miinitext.o     \
 XPMLIB    =  -lXpm
 XMLLIB    =  `pkg-config --libs libxml-2.0`
 PIXMANLIB =  `pkg-config --libs pixman-1`
-ZLIB      =  `pkg-config --libs zlib`
+ZLIB      =  `pkg-config --libs-only-l zlib`
```

Comment:
After your linker options order change patch, at least.

I'm not sure if github gets the commit order right, but doesn't it work with the linker options order change first and this commit dropped completely?